### ISSUE TITLE
Maui project FloatingMenu not work for android.

### ISSUE THREAD

**Issue by iwaitu** [UXDivers/Grial-UI-Kit-Support#861]
I downloaded the project, build and run the project on android emnulator.

When I try to open the floatingmenu in ExplorerFiles Page ,the app was crashed.

**Comment by Santiago5050** [comment]
Hi @iwaitu,

This issue was fixed in our last release. Please try updating the grial nuget to the last version (4.1.106).

Regards,
Santiago.

**Comment by iwaitu** [comment]
When I updated UXDivers.GrialMaui to 4.1.106,and UXDivers.GrialMaui.Maps to 4.1.105, I found other probelm. xmlns:grialMaps="clr-namespace:UXDivers.Grial;assembly=UXDivers.GrialMaui.Maps" this line throw error . map controller was not in the namespace . please notice us if you changed the namespace.

**Comment by Santiago5050** [comment]
Yes, we change the namespace for the maps, now you can define the new namespace like this:

`xmlns:grialMaps="clr-namespace:UXDivers.Grial.Maps;assembly=UXDivers.GrialMaui.Maps"`

Also, you need to add the next using to the MauiProgram to set up the maps without errors:

`using UXDivers.Grial.Maps;`

Regards,
Santiago.
